### ISSUE TITLE
GCop302 false positive

### ISSUE THREAD

**Issue by HamidTheGeek** [Geeksltd/GCop#310]
A [new syntax is introduced with C# 8](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations) and it is the recommended way. But, GCop warns about it.

```C#
using var command = new SqlCommand(ADD_COMMAND_C, connection);
```
instead of
```C#
using(var command = new SqlCommand(ADD_COMMAND_C, connection))
{
     ...
}
```
